packages/core/src/errors.ts

```diff
@@ -77,6 +77,7 @@ export const enum RuntimeErrorCode {
   INVALID_SKIP_HYDRATION_HOST = -504,
   MISSING_HYDRATION_ANNOTATIONS = -505,
   HYDRATION_STABLE_TIMEDOUT = -506,
+  MISSING_HYDRATION_MARKER_COMMENT = 507,
```

Comment:
For consistency, let's use the same name here as well:
```suggestion
  MISSING_SSR_CONTENT_INTEGRITY_MARKER = 507,
```